File: 344-ReverseString/344-ReverseString.py
# Last updated: 10/3/2025, 2:13:27 PM
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        i=0
        j=len(s)-1

        while(i<j):
            t=s[i]
            s[i]=s[j]
            s[j]=t
            i+=1
            j-=1
        return s

File: 344-ReverseString/344-ReverseString.java
// Last updated: 10/3/2025, 2:13:31 PM
class Solution {
    public void reverseString(char[] c) {
        int p1=0;
        int p2=c.length-1;

        while(p1<p2){
            char temp=c[p1];
            c[p1]=c[p2];
            c[p2]=temp;
            p1++;p2--;
        }
    }
}